tools/actool.mk

```diff
@@ -4,6 +4,7 @@ $(call setup-stamp-file,ACTOOL_STAMP)
 BGB_STAMP := $(ACTOOL_STAMP)
 BGB_PKG_IN_REPO := Godeps/_workspace/src/github.com/appc/spec/actool
 BGB_BINARY := $(ACTOOL)
+BGB_ADDITIONAL_GO_ENV := GOARCH=$(GOARCH_FOR_BUILD) CC=$(CC_FOR_BUILD)
```

Comment:
Is overriding `CC` env var required? I think that `go` should be able to pick up a valid C compiler based on `GOARCH` if it needs to build some C sources too…